examples/simple/variables.tf

```diff
@@ -1,21 +1,14 @@
-# TEMPLATE: All variables must have a description and should declare their type.
-# TEMPLATE: Set defaults whenever possible but do not set defaults for required properties.
-# TEMPLATE: Declare all variables in this file, sprawling declarations are difficult to identify.
-# TEMPLATE:
-# TEMPLATE: https://www.terraform.io/docs/language/values/variables.html
-# TEMPLATE: https://www.terraform.io/docs/language/expressions/types.html
-# TEMPLATE:
+variable "runner_scope" {
+  type        = string
+  description = "To specify repositories, use the format ':owner/:repo'. For example, 'equinix-labs/terraform-equinix-github-runner'. To specify an organization, use the format ':organization'"
+}
 
-# TEMPLATE: Replace sample variable described below with your own.
-variable "example_metal_project_id" {
+variable "personal_access_token" {
   type        = string
-  description = "The example project id value defines what will be included in the example resource in main.tf. This example is descriptive."
-  sensitive   = false
+  description = "GitHub PAT (Personal Access Token)"
```

Comment:
Could mark the variable `sensitive` here rather than using the `sensitive` function.

Reply:
N/A